xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -4008,7 +4008,7 @@ xe.admin.passwordreset.nouser=The ~~{0}~~ user does not exist.
 xe.admin.passwordreset.ldapuser=The ~~{0}~~ user is an LDAP user. In that case the password has to be changed on the LDAP server.
 xe.admin.passwordreset.cannotreset=Cannot reset password: email address not provided in the user profile.
 xe.admin.passwordreset.emailsent=An e-mail was sent to <tt>{0}</tt>. Please follow the instructions in that e-mail to complete the password reset procedure.
-xe.admin.passwordreset.reseterror=An unknown problem occurred while sending the reset email.
+xe.admin.passwordreset.reseterror=An unknown problem occurred while sending the reset e-mail.
```

Comment:
In my opinion, we should keep `email` (without hyphen) as we are using this word across all of our translations.
Moreover, it seems that `email` should be preferred against `e-mail` according to [this source](http://grammarist.com/style/e-mail-email/).